client/my-sites/upgrades/index.js

```diff
@@ -284,10 +285,19 @@ module.exports = function() {
 		);
 
 		page(
-			'/checkout/:domain/:plan_name?',
+			'/checkout/:domain/:planName?',
 			adTracking.retarget,
 			controller.siteSelection,
 			upgradesController.checkout
 		);
+
+		page(
+			'/start-trial/:planName/:domain',
```

Comment:
I don't feel strongly about `:domain` but it is used everywhere for paths in this file https://github.com/Automattic/wp-calypso/blob/update/free-trial-checkout-page/client/my-sites/upgrades/index.js
So I kept it for consistency.